AncientReligions/localisation/ACR_hellenic_events.csv

```diff
@@ -29,6 +30,18 @@ EVTNAMEancrel.2104;Return of the Pontificate Maxima;;;;;;;;;;;;;x
 EVTDESCancrel.2104;After an interlude lasting a few centuries following the adoption of Christianity in Rome, Hellenism has returned as the state religion of [From.PrimaryTitle.GetName] and [From.GetTitledFirstName] has ordered the reinstitution of the Collegium Pontificum.\n\nThe Pontificate Maxima will serve as the highest religious authority in the Hellenic world, with powers ranging from the blessing of future military expeditions to the nomination of Vestals and flamens.;;;;;;;;;;;;;x
 EVTOPTAancrel.2104;Rejoice!;R�jouissons-nous !;Frohlocket!;;�Regocijaos!;;;;;;;;;x
 EVTOPTBancrel.2104;Heathens!;Pa�ens !;Ketzer!;;�Paganos!;;;;;;;;;x
+###### Mani Penninsula;;;;;;;;;;;;;;x
+EVTDESCancrel.2105;You have heard rumours about a group of Hellenic peasants on the Mani Penninsula who continue to practice their old ways. There is much debate at court on how large that group is, and how much their practices may have evolved since the old times...;;;;;;;;;;;;;x
+EVTOPTAancrel.2105;This is an opportunity to better know my realm.;;;;;;;;;;;;;x
+EVTOPTBancrel.2105;How stupid, the Hellenic faith is long dead !;;;;;;;;;;;;;x
+travel_penninsula;You travel to the penninsula\n;;;;;;;;;;;;;x
+EVTDESCancrel.2106;Reaching the tip of the penninsula, you discover in the hills a waterless and innaccesible place where the Maniots live. As you approach, you notice an old [Root.target_maniot_priest.GetManWoman] who seem to be waiting for you. [Root.target_maniot_priest.GetSheHeCap] explains that the gods have sent signs of your venue.;;;;;;;;;;;;;x
```

Comment:
Missing an 's' at the end of 'seems' and I think you meant "arrival" instead of "venue."